probe_scraper/runner.py

```diff
@@ -518,21 +492,56 @@ def load_glean_metrics(
             if glean_repos is None and glean_urls is None:
                 raise
 
+    # currently always true, but left in for clarity
+    if scrape_commits or generate_metadata:
```

Comment:
and future bug resistance, should there become a case where this overlap isn't always true